src/current/v23.1/import-performance-best-practices.md

```diff
@@ -7,26 +7,89 @@ docs_area: migrate
 
 This page provides best practices for optimizing [import](import-into.html) performance in CockroachDB.
 
-Import speed primarily depends on the amount of data that you want to import. However, there are two main factors that have can have a large impact on the amount of time it will take to run an import:
+`IMPORT INTO` is the fastest method to ingest data into CockroachDB but it requires taking the target table offline for the duration of the import. `IMPORT INTO` is a good choice for initial data migrations and data migrations that can tolerate table downtime. If you cannot tolerate table unavailability, we recommend using [`COPY FROM`](copy-from.html) instead.
 
-- [Splitting data](#split-your-data-into-multiple-files)
-- [Import format](#choose-a-performant-import-format)
+Import performance primarily depends on the amount of data that you want to import. However, there are three actions you can take before importing that have a significant impact on the amount of time it will take to run an import:
+
+- [Choose a performant import format](#choose-a-performant-import-format)
+- [Split your data into multiple files](#split-your-data-into-multiple-files)
+- [Sort your data](#sort-your-data)
 
 {{site.data.alerts.callout_info}}
-If the import size is small, then you do not need to do anything to optimize performance. In this case, the import should run quickly, regardless of the settings.
+If the import size is small, then you do not need to do anything to optimize performance. For small datasets, the import should run quickly, regardless of the settings.
 {{site.data.alerts.end}}
 
+## Choose a performant import format
+
+Different import file formats do not have the same performance due to the way they are processed by CockroachDB. The fastest import file formats are:
+
+1. [`CSV`](migrate-from-csv.html) or [`DELIMITED DATA`](import-into.html) (both have about the same import performance).
+1. [`AVRO`](migrate-from-avro.html).
+
+We recommend formatting your import files as `CSV` or `AVRO`. These formats can be processed in parallel by all the nodes in the cluster, which increases performance. To import in these formats, use [`IMPORT INTO`](import-into.html).
+
+{% include {{ page.version.version }}/import-table-deprecate.md %}
+
+### Import the schema separately from the data
+
```

Comment:
This only works for importing into a new table, correct? If Import into an existing table, the schema is already set. If my understanding is correct, shall we call it out?